microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -30,10 +30,10 @@ An overview of the changes involved, as described in [Overview of Manifest V3 -
 
 To help you plan, consider the following schedule for Microsoft Partner Center and Microsoft Edge changes.
 
+We are currently in the process of updating our MV3 migration timelines.
 
-#### Revised timeline
+As of now, we will continue to follow Chromium’s MV3 timelines. While we fundamentally align with these timelines, we will continue to analyze the concerns raised by extension developers and explore the right path for Microsoft Edge Addons ecosystem as a whole. We will independently decide on MV3 migration timelines for Microsoft Edge Addons and share an update here once they are available.
 
-This timeline is subject to change. This article will be updated as the milestones draw nearer.  Chrome has revised the timeline for Manifest V2 sunset, and we are about to update this article to reflect that change.
```

Comment:
Please change the January 2023 and June 2023 timelines in the table to "TBD - see note above"